addons/l10n_tr_nilvera/data/uom_data.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
```

Comment:
I think you need to keep the category (or create an update script to deal with an existing one in 18.0)

Reply:
The model was entirely removed so i don't know if i really need to do something ?

Reply:
aaaaah sorry then :) 
The upgrade script should exist already